dev/lib/product_taxonomy/models/attribute.rb

```diff
@@ -42,6 +42,11 @@ def sorted_base_attributes
         all.reject(&:extended?).sort_by(&:name)
       end
 
+      # Get the next ID for a newly created attribute.
+      #
+      # @return [Integer] The next ID.
+      def next_id = (all.map(&:id).max || 0) + 1
```

Comment:
Missed this in previous reviews, but I think we can be more efficient:

```rb
def next_id = (all.max_by(&:id).id || 0) + 1
```

I do wish we could have done something like `all.max(&:id)` but not supported syntax :shakes-fist:

Reply:
Nice 👍 I updated this on `Attribute` and also `Value`